tests/autologging/test_autologging_utils.py

```diff
@@ -657,8 +657,8 @@ def test_check_version_in_range():
 
 
 @pytest.mark.parametrize("flavor,module_version_dict,expected_result", [
-    ("fastai", {"fastai": "1.0.60"}, True),
-    ("fastai", {"fastai": "1.0.50"}, False),
+    ("fastai", {"fastai": "1.0.30"}, True),
+    ("fastai", {"fastai": "1.0.20"}, False),
```

Comment:
With the current approach, we'd need to make update the tests when updating `mlflow/ml-package-versions.yml`.

Reply:
No. minimal version is fixed. so I only test against patch version < min version and in range  cases.